posts/deep_work.md

```diff
@@ -0,0 +1,35 @@
+# How deep is your work?
+
+Coined by Cal Newport in 2012 on his Study Hacks blog, ‘[deep work](http://calnewport.com/books/deep-work/)’ is the ability to reach a state of increased productivity when performing cognitively taxing tasks by minimising or ignoring external interferences. 
+Making deep work the centrepoint of your knowledge work schedule, generates three key benefits:
+1. An increase in the quality of your output;
+2. An increase in the quantity of your output; and 
+3. A passion that manifests as a deeper sense of satisfaction when undertaking and completing tasks.
+
+In a 24/7 world driven by technological change and innovation, distractions are everywhere and unavoidable. Emails, social media interaction and digitalisation of basic administrative tasks often require immediate attention and an instantaneous response. The workspace, whether office-based or remote, as a result, becomes a hot-bed of multi-tasking, interruption, and profound information overload. And when it’s been [found](https://www.ics.uci.edu/~gmark/chi08-mark.pdf) that re-focusing on a task after being distracted takes a person on average, 23 minutes and 15 seconds, this just leads to more speed and more stress. 
+
+Like a lock that eventually gives way after being repeatedly picked at, unconstructive, fragmented time reduces the ability to concentrate and therefore resist such distractions. If permitted to continue then over time, deeper working becomes harder to attain and you need to work for an increasing amount of time on complex tasks before reaching peak productivity.
+
+Consequently, at Made Tech, we’ve been trying a number of techniques to help teams commit to periods of deep working. Success of each varies from person-to-person as everyone responds differently to external stimuli and some are naturally better than others at zoning out distractions. While it’s worth taking time to work out what works on an individual level, these are some strategies we would recommend. 
+
+## Adapt your working environment
+
+Flexible working is embedded in our culture so people can choose where they work from and largely when. We do however try to discourage working in the evenings to ensure work/life balance, and because it has the potential to prevent relaxation and risk negatively impacting work being done the following day.     
+
+## Time yourself
+
+During pairing activities our teams use [pomodoros](http://tomatotimers.com/) - a time management method which breaks down work into intervals of 25 minutes in length, separated by short five minute breaks.  Our developers and engineers have also used the timer system while working on client projects with good outcomes, so recently we extended its usage to the marketing and sales teams as well. 
```

Comment:
I think this is "officially" called the pomodoro technique (there's a wikipedia article at least!). I think often pomos vary in length too - there's no hard and fast 25 minute

Reply:
Unclear on this point about "also using the system on client projects"

Reply:
I think we could make clearer that this activity isn't limited to development or pairing environments - anything that requires periods of uninterrupted concentration can benefit